tests/test_aws.py

```diff
@@ -43,6 +43,9 @@ def test_aws_scan(self):
     with open(results_dir.joinpath(f"aws-cost-{sub_id}.json")) as f:
         costs = json.load(f)
         assert len(costs["data"]) >= 100
+        for cost in costs["data"]:
```

Comment:
use a fixture for this test. I don't think we have a savings plan with AWS

Reply:
What should I use as the fixture? is there an `aws-cost-foo.json` that I can use?